### PR TITLE
impl Send and Sync for IoVec

### PR DESCRIPTION
**Pull request by asomers**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## [Unreleased] - ReleaseDate
 
 ### Added
+
+- `sys::uio::IoVec` is not `Send` and `Sync`
```
Comment (rtzoeller) [inline CHANGELOG.md:10]: now -> now?

Comment (asomers) [reply CHANGELOG.md]: fixed

**Comment by rtzoeller** [comment]
@asomers are you intending this for 0.23.1 or 0.24.0?

**Comment by asomers** [comment]
The latter

**Comment by rtzoeller** [comment]
@asomers this needs a rebase for the changelog.

**Pull request merged**